### PR TITLE
[#1032] Advanced settings

### LINKED ISSUE #1032: Advanced settings
The settings screen is overloaded so we decided to split it and put some advanced options onto a whole new screen.

### LINKED ISSUE #1033: Privacy policy link in About screen
The privacy policy has been removed from the settings as a standalone button and was put into an About screen. 

https://www.figma.com/file/vYQD6uZtkPgtmgJ3zfauq0/Zashi-UI?type=design&node-id=149-7969&mode=design&t=L69gPR22NMvtqLNn-0

### PR DESCRIPTION
**Pull request by LukasKorba**
Closes #1032
Closes #1033 

- all buttons moved to appropriate positions and screens
- business logic rewired
- all new texts localized
- changelog updated
- unit tests fixed

Simulator Screenshot - iPhone 15 Pro - 2024-02-12 at 13 08 49
Simulator Screenshot - iPhone 15 Pro - 2024-02-12 at 13 08 51

This code review checklist is intended to serve as a starting point for the author and reviewer, although it may not be appropriate for all types of changes (e.g. fixing a spelling typo in documentation).  For more in-depth discussion of how we think about code review, please see [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md).

# Author

- [x] Self-review: Did you review your own code in GitHub's web interface? Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs.
- [ ] Does the code abide by the [Coding Guidelines](../blob/main/docs/CODING_GUIDELINES.md)?
- [ ] Automated tests: Did you add appropriate automated tests for any code changes?
- [x] Code coverage: Did you check the code coverage report for the automated tests?  While we are not looking for perfect coverage, the tool can point out potential cases that have been missed.
- [ ] Documentation: Did you update Docs as appropiate? (E.g [README.md](../blob/main/README.md), etc.)
- [x] Run the app: Did you run the app and try the changes? 
- [x] Did you provide Screenshots of what the App looks like before and after your changes as part of the description of this PR? (only applicable to UI Changes)
- [ ] Rebase and squash: Did you pull in the latest changes from the main branch and squash your commits before assigning a reviewer? Having your code up to date and squashed will make it easier for others to review. Use best judgement when squashing commits, as some changes (such as refactoring) might be easier to review as a separate commit.

# Reviewer

- [ ] Checklist review: Did you go through the code with the [Code Review Guidelines](../blob/main/CODE_REVIEW_GUIDELINES.md) checklist?
- [ ] Ad hoc review: Did you perform an ad hoc review?  _In addition to a first pass using the code review guidelines, do a second pass using your best judgement and experience which may identify additional questions or comments. Research shows that code review is most effective when done in multiple passes, where reviewers look for different things through each pass._
- [ ] Automated tests: Did you review the automated tests?
- [ ] Manual tests: Did you review the manual tests?_You will find manual testing guidelines under our [manual testing section](../blob/main/docs/testing/manual_testing)_
- [ ] How is Code Coverage affected by this PR? _We encourage you to compare coverage before and after your changes and when possible, leave it in a better place. [Learn More...](../blob/main/docs/testing/local_coverage.md)_
- [ ] Documentation: Did you review Docs, [README.md](../blob/main/README.md), [LICENSE.md](../blob/main/LICENSE.md), and [Architecture.md](../blob/main/docs/Architecture.md) as appropriate?
- [ ] Run the app: Did you run the app and try the changes? While the CI server runs the app to look for build failures or crashes, humans running the app are more likely to notice unexpected log messages, UI inconsistencies, or bad output data.

### CODE REVIEW

File: modules/Sources/Features/Settings/SettingsStore.swift [hunk 1]
```diff
@@ -24,61 +18,46 @@ public struct SettingsReducer: Reducer {
...
 
     public enum Action: Equatable {
+        case advancedSettings(AdvancedSettingsReducer.Action)
         case alert(PresentationAction<Action>)
-        case backupWalletAccessRequest
         case onAppear
-        case phraseDisplay(RecoveryPhraseDisplayReducer.Action)
-        case privateDataConsent(PrivateDataConsentReducer.Action)
         case restoreWalletTask
         case restoreWalletValue(Bool)
```
Comment (str4d) [inline modules/Sources/Features/Settings/SettingsStore.swift:54]: Are these still needed here? I see they were added to `AdvancedSettingsStore`m like other enum variants that were removed from here.

File: modules/Sources/Features/Settings/SettingsStore.swift [hunk 2]
```diff
@@ -24,61 +18,46 @@ public struct SettingsReducer: Reducer {
...
         case sendSupportMail
         case sendSupportMailFinished
-        case serverSetup(ServerSetup.Action)
         case updateDestination(SettingsReducer.State.Destination?)
```
Comment (str4d) [inline modules/Sources/Features/Settings/SettingsStore.swift:57]: Ditto, is this variant needed here now that it is in `AdvancedSettingsStore`?

**Review by str4d — APPROVED** [review]
utACK 0cf120b8c32c31d67f1c74f00cc25542468f8c6a. The first commit in the PR I reviewed in #1044, and IDK what the second commit is for (it has the same commit message as the first, but seemingly unrelated changes). Standard disclaimer that I Am Not A Swift Or iOS Developer.

File: modules/Sources/Features/Settings/SettingsStore.swift (hunk 1 above)
Comment (LukasKorba) [reply modules/Sources/Features/Settings/SettingsStore.swift:54]: Both are used for a different thing unrelated to this ticket. It's about this restore UI. The actions are used to subscribe to the service that updates the state. The UI is rendered on all possible screens so both Settings and AdvancedSettings present it.

Screenshot 2024-02-14 at 11 13 56

File: modules/Sources/Features/Settings/SettingsStore.swift (hunk 2 above)
Comment (LukasKorba) [reply modules/Sources/Features/Settings/SettingsStore.swift:57]: Both are used for a different thing unrelated to this ticket. It's about this restore UI. The actions are used to subscribe to the service that updates the state. The UI is rendered on all possible screens so both Settings and AdvancedSettings present it.

Screenshot 2024-02-14 at 11 13 56

**Pull request merged**